mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -78,7 +78,7 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 	FIELD field_6029 scoreboardTags Ljava/util/Set;
 	FIELD field_6030 EMPTY_STACK_LIST Ljava/util/List;
 	FIELD field_6031 yaw F
-	FIELD field_6032 BREATH Lnet/minecraft/class_2940;
```

Comment:
Given this applies to water mobs, breath may actually be more accurate

Reply:
Breath isn't a good name for a thing that's serialized to `Air`.

Reply:
Mojang cannot change their serialized names easily due to backward compatibility, so they might have left it as-is in nbt but changed it in their actual code.

Reply:
> Mojang cannot change their serialized names easily due to backward compatibility

They can, though? They can make a datafixer that targets entities.